### PR TITLE
Jetpack: Remove monthly plans test

### PR DESCRIPTION
**Pull request by johnHackworth**
Fixes https://github.com/Automattic/wp-calypso/issues/11607

TLDR: We are going to run an a/b test removing the jetpack monthly plans.

This PR implements it.

How to test:
0. First, go to http://calypso.localhost:3000/plans/ and select a jetpack site ... make sure everything looks right. Then connect that site and make sure everything works fine in the plans selection screen ( you will need a site without any purchased plan so you can see the plans step there)
1. Now let's start the test. Open the developer tools and run this in the console: `localStorage.setItem( 'ABTests', '{"jetpackPlansNoMonthly_20170302":"hideMonthly"}' );`
2. Go to http://calypso.localhost:3000/plans/ and select a jetpack site that doesn't have any paid plan. You should see something like this:
image

3. Disconnect that site and connect it again. In the plans selection step you should see:
image

4. Now purchase a plan with that site (or select any site with an already purchased plan). Go back to  http://calypso.localhost:3000/plans/[site-slug]. This time, you should see the monthly plans selector: 
image
Make sure you can see the monthly version of the plans

5. Now choose a WordPress.com site and go to the plans page again. Make sure nothing in this PR interfers with dotcom plans

### CODE REVIEW

File: client/lib/abtest/active-tests.js [hunk 1]
```diff
@@ -113,4 +103,15 @@ module.exports = {
...
 		allowExistingUsers: true
 	},
+
+	jetpackPlansNoMonthly: {
+		datestamp: '20170302',
+		variations: {
+			showMonthy: 50,
+			hideMonthly: 50
+		},
+		defaultVariation: 'showMonthy',
```
Comment (tyxla) [inline client/lib/abtest/active-tests.js]: We have a typo on various locations: `showMonthy` instead of `showMonthly`.

Comment (johnHackworth) [inline client/lib/abtest/active-tests.js]: damn autocomplete ... make the typo once, propagate it forever

**Comment by johnHackworth** [comment]
good catch @tyxla . I handled that case in a new commit
